alfred-telemetry-solr/alfred-telemetry-solr4/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrMetrics.java

```diff
@@ -0,0 +1,35 @@
+package eu.xenit.alfred.telemetry.solr.monitoring.binder;
+
+import eu.xenit.alfred.telemetry.solr.util.Util;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import org.alfresco.solr.AlfrescoCoreAdminHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.MBeanServer;
+
+public class SolrMetrics implements MeterBinder {
+
+    AlfrescoCoreAdminHandler coreAdminHandler;
+    MBeanServer mBeanServer;
+
+    Logger logger = LoggerFactory.getLogger(SolrMetrics.class);
+
+    public SolrMetrics(AlfrescoCoreAdminHandler coreAdminHandler, MBeanServer mBeanServer) {
+        this.coreAdminHandler = coreAdminHandler;
+        this.mBeanServer = mBeanServer;
+    }
+
+    @Override
+    public void bindTo(MeterRegistry registry) {
+        if(Util.isEnabled("METRICS_SOLR_CORESTATS_ENABLED"))
```

Comment:
Use square braces in `if` statements 

https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used